.github/workflows/evergreen.yaml

```diff
@@ -41,6 +42,7 @@ jobs:
       platform: evergreen-arm-hardfp
       nightly: ${{ github.event.inputs.nightly }}
       run_api_leak_detector: true
+      check_artifact_size: evergreen.yaml
       keep_artifacts: libcobalt.so
```

Comment:
Would it be a lot of work to update the check to also check libcobalt.lz4?

Reply:
To check more than one artifact per workflow, we will need to refactor both the existing `Upload Artifact` action and the new`Check Artifact Size` action to take in an array of artifacts. Should be a small to medium size PR. I can do a follow-up PR.